src/kallisto/kallisto_index/help.txt

```diff
@@ -0,0 +1,21 @@
+```
+kallisto index
+```
+kallisto 0.50.1
+Builds a kallisto index
+
+Usage: kallisto index [arguments] FASTA-files
+
+Required argument:
+-i, --index=STRING          Filename for the kallisto index to be constructed 
+
+Optional argument:
+-k, --kmer-size=INT         k-mer (odd) length (default: 31, max value: 31)
```

Comment:
This is argument is not used which will lead to poor performance because of the default being 1.

Please take a look [here](https://viash.io/guide/component/variables.html#cpus-integer) and [here](https://github.com/viash-hub/biobox/blob/320d044fe45e565fbc9772640ebf6f39c5584b4a/src/busco/busco_run/script.sh#L34).

Reply:
Right, since the contributing guidelines said this "Arguments related to specifying CPU and memory requirements are handled separately and should not be added to the config file.", I assumed I shouldn't include it in the script but I get how that doesn't make sense! :)